examples/basic/annotations/legend_name.py

```diff
@@ -0,0 +1,15 @@
+from bokeh.models import Legend, LinearAxis, Range1d
+from bokeh.plotting import figure, save
```

Comment:
@mattpap You used `save` instead of `show`. I guess this is just a typo. If you agree this is a bug, I will open a new issue.